tsl/test/expected/hypercore_vectoragg.out

```diff
@@ -0,0 +1,250 @@
+select format('%I.%I', chunk_schema, chunk_name)::regclass as chunk
+from timescaledb_information.chunks
+where hypertable_name='aggdata'
+limit 1 \gset
+alter table aggdata set (timescaledb.compress_orderby='time', timescaledb.compress_segmentby='device');
+alter table :chunk set access method hypercore;
+-- Add some non-compressed data to ensure vectoraggs work with both
+-- compressed and non-compressed data.
+insert into aggdata values ('2024-01-01 02:00', 1, 1, 3.0);
+--
+-- Run a number of queries to compare plans and output with vectorized
+-- aggregation on and off.
+--
+-- This is just to do basic checks to ensure VectorAggs produce the
+-- expected output.
+--
+-- To run on larger data, the vectorized_aggregation test can be run
+-- with TAM enabled by default.
+--
+explain (costs off)
+select avg(device) from aggdata;
+                            QUERY PLAN                            
+------------------------------------------------------------------
+ Finalize Aggregate
+   ->  Append
+         ->  Custom Scan (VectorAgg)
+               ->  Custom Scan (ColumnarScan) on _hyper_1_1_chunk
+         ->  Partial Aggregate
+               ->  Seq Scan on _hyper_1_2_chunk
+(6 rows)
+
+select avg(device) from aggdata;
+        avg         
+--------------------
+ 1.7500000000000000
+(1 row)
+
+set timescaledb.enable_vectorized_aggregation=false;
+explain (costs off)
+select avg(device) from aggdata;
+                            QUERY PLAN                            
+------------------------------------------------------------------
+ Finalize Aggregate
+   ->  Append
+         ->  Partial Aggregate
+               ->  Custom Scan (ColumnarScan) on _hyper_1_1_chunk
+         ->  Partial Aggregate
+               ->  Seq Scan on _hyper_1_2_chunk
+(6 rows)
+
+select avg(device) from aggdata;
+        avg         
+--------------------
+ 1.7500000000000000
+(1 row)
+
+set timescaledb.enable_vectorized_aggregation=true;
+explain (costs off)
+select avg(temp) filter (where device > 1) from aggdata;
+                            QUERY PLAN                            
+------------------------------------------------------------------
+ Finalize Aggregate
+   ->  Append
+         ->  Custom Scan (VectorAgg)
+               ->  Custom Scan (ColumnarScan) on _hyper_1_1_chunk
+         ->  Partial Aggregate
+               ->  Seq Scan on _hyper_1_2_chunk
+(6 rows)
+
+select avg(temp) filter (where device > 1) from aggdata;
+ avg 
+-----
+ 2.5
+(1 row)
+
+set timescaledb.enable_vectorized_aggregation=false;
+explain (costs off)
+select avg(temp) filter (where device > 1) from aggdata;
+                            QUERY PLAN                            
+------------------------------------------------------------------
+ Finalize Aggregate
+   ->  Append
+         ->  Partial Aggregate
+               ->  Custom Scan (ColumnarScan) on _hyper_1_1_chunk
+         ->  Partial Aggregate
+               ->  Seq Scan on _hyper_1_2_chunk
+(6 rows)
```

Comment:
I don't see the filter in the plan anywhere. Can we use `verbose` so that that we show the filter and are sure that it has not been lost because of a programming error?

Reply:
Added verbose.